js/rank_states_utils.js

```diff
@@ -135,17 +136,36 @@ function rankEm() {
           .transition().duration(600)
             .attr('transform','translate(0,0)'); 
       } else {
-        barchoice
-          .on('mouseover', null)
-          .on('mouseout', null)
-          .attr('class','closed-rank-bar');
-        d3.select(barchoice.node().parentNode).select('text')
-          .classed('open-bar-name',false);
-        d3.select(this).style('opacity', 0)
-          .attr('transform','scale(0)'); 
+        
+        var thisBarname = d3.select(barchoice.node().parentNode).select('text').text();
+        
+        if (thisBarname + '-rank' == d3.select(this).attr('id')){
+          // the guess is right
+           barchoice
+            .on('mouseover', null)
+            .on('mouseout', null)
+            .attr('class','closed-rank-bar');
+          d3.select(barchoice.node().parentNode).select('text')
+            .classed('open-bar-name',false);
+          d3.select(this).style('opacity', 0).attr('transform','scale(0)'); 
+          // one last time in cast it overlapped two rectangles
+          overRankBar(d3.select(this).node().getBoundingClientRect()); 
+          
+        } else {
+          barchoice.attr('class','open-rank-bar');
+          barchoice.transition().duration(600).ease(d3.easeLinear)
+            .style('stroke-dashoffset',"6")
+            .style('stroke', 'rgb(200, 65, 39)') // emphasize with a redorange stroke
```

Comment:
in the gif you included, that color didn't come through obviously. Should we fill the bar to make it more obvious? We could also have it flash redorange fill and then back to nothing.

Reply:
Yeah, it isn't super obvious, but I didn't want to go too far messing around with strokes and classes since I thought we still might make some changes as to how that is organized. I am not very satisfied with my mix of `.style` and `.classed` in here, but I think it is OK for now.

Reply:
oh yeah, and it does look like the gif colors made my redorange look darker than it appears on screen